modules/apps/fragment/fragment-collection-contributor/fragment-collection-contributor-basic-component/src/main/resources/com/liferay/fragment/collection/contributor/basic/component/dependencies/tabs/index.js

```diff
@@ -32,11 +32,18 @@ const persistedTab = (function () {
 
 	return {
 		getId() {
-			return Number(sessionStorage.getItem(persistedTabKey));
+			return Number(
+				Liferay.Util.SessionStorage.getItem(persistedTabKey),
+				Liferay.Util.COOKIE_TYPES.PERSONALIZATION
+			);
```

Comment:
Does this do anything? `Number` has a single argument, the second one is ignored.